install/kubernetes/helm/meshery/charts/meshery-nighthawk/Chart.yaml

```
@@ -0,0 +1,33 @@
+apiVersion: v2
+name: meshery-nighthawk
+description: Meshery Adapter for App Mesh chart.
+icon: https://meshery.io/images/logos/meshery-logo.png
+source:
+  - https://github.com/layer5io/meshery-nighthawk
+
+deprecated: false
+
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+version: 0.7.0
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application.
+# TODO: Need the confirm the image tag.
+appVersion: stable-latest
```

```suggestion
appVersion: edge-latest
```
We only have the edge-latest version of meshery-nighthawk image currently: https://hub.docker.com/layers/meshery/meshery-nighthawk/edge-latest/images/sha256-4efc9a803b2c31b0395596af6bc4b1289da067fc058684cbf49799eb56e74901?context=explore